mimic/rest/heat_api.py

```diff
@@ -138,9 +141,39 @@ def extract_tags():
             tags=extract_tags(),
             absolutize_url=self.url)
 
-    @app.route(
-        '/v1/<string:tenant_id>/stacks/<string:stack_name>/<string:stack_id>',
-        methods=['DELETE'])
+    @app.route(STACK_URL, methods=['PUT'])
+    def update_stack(self, request, tenant_id, stack_name, stack_id):
+        """
+        Updates a stack.
+        See http://api.rackspace.com/api-ref-orchestration.html#stack_update
+        """
+        region_collection = self._region_collection_for_tenant(tenant_id)
+        return region_collection.request_update(request, stack_name, stack_id)
+
+    @app.route(STACK_URL + '/actions', methods=['POST'])
+    def perform_action(self, request, tenant_id, stack_name, stack_id):
+        """
+        Performs an action on a stack. Only action-check is enabled at
+        Rackspace.
+        See http://api.rackspace.com/api-ref-orchestration.html#stack_update
```

Comment:
This links to stack update doc. Moreover, I couldn't find actions doc in api.rackspace.com or developer.rackspace.com. I wonder if you should link to http://developer.openstack.org/api-ref-orchestration-v1.html#stack-actions.